subprojects/android-lib/proxy-toast/build.gradle.kts

```diff
@@ -1,10 +1,9 @@
 plugins {
-    id("com.android.library")
-    id("kotlin-android")
-    `maven-publish`
-    id("com.jfrog.bintray")
+    id("com.avito.android.kotlin-android-library")
+    id("com.avito.android.libraries")
+    id("com.avito.android.publish")
 }
 
 dependencies {
-    implementation(Dependencies.androidAnnotations)
+    implementation(libs.androidAnnotations)
```

Comment:
What do you not like about using `Dependencies`

Reply:
please take a look on MBS-10605 task description